app/queue/egress.py

```diff
@@ -87,8 +95,13 @@ class HostSchema(Schema):
     system_profile = fields.Nested(SystemProfileSchema)
 
 
+class metadataSchema(Schema):
+    request_id = fields.Str()
+
+
 class HostEvent(Schema):
     type = fields.Str()
     host = fields.Nested(HostSchema())
     timestamp = fields.DateTime(format="iso8601")
     platform_metadata = fields.Dict()
+    metadata = fields.Nested(metadataSchema())
```

Comment:
This schema would be universal for both the delete event and the create/update event. Remove _metadata_ if request ID in root is ok.

```suggestion
class HostEvent(Schema):
    type = fields.Str()
    host = fields.Nested(HostSchema())
    timestamp = fields.DateTime(format="iso8601")
    platform_metadata = fields.Dict()
    insights_id = fields.Str()
    request_id = fields.Str()
    metadata = fields.Nested(HostEventMetadataSchema())
```